### ISSUE TITLE
Fix documentation of default thread pool size

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8104]
|<img alt="bkambach's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| bkambach reported|
|-|-|
|Trac ID|trac#8104|
|Type|defect|
|Created|2015-11-12 18:25:34Z|
|Branch|https://github.com/twisted/twisted/tree/doc-howto-threading-pool-size-8104|

The default reactor is described as having a minimum threadpool size of 5 (​https://twistedmatrix.com/documents/current/core/howto/threading.html#managing-the-reactor-thread-pool):
>The default size of the thread pool depends on the reactor being used; the default reactor uses a minimum size of 5 and a maximum size of 10.

However, as noted in comment:9:ticket:8090, this is incorrect as the minimum is actually 0.

Attachments:

* [threading-docfix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/277/277aac589bd8f21291ee41fef067154ce3f420c2/3fd4732e4e32273fa3360712ffff646919274397.patch) (886 bytes) - added by thijs on 2015-11-18 16:06:40Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__8104 8104
type__defect defect
reporter__bkambach bkambach
priority__normal normal
milestone__None None
branch__branches_doc_howto_threading_pool_size_8104 branches/doc-howto-threading-pool-size-8104
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1447352734876174 1447352734876174
changetime__1447958778978371 1447958778978371
version__None None
owner__thijs thijs
cc__oubiwann cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

(In [46264]) Branching to doc-howto-threading-pool-size-8104.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @thijstriemstra|
|-|-|

Many thanks for the fix.

Changes applied. Waiting for tests results and will merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46294]) Merge doc-howto-threading-pool-size-8104: Fix documentation of default thread pool size.

Author: thijs
Reviewer: adiroiban
Fixes: [#8104](https://github.com/twisted/twisted/issues/8104)
